libs/ui/src/components/NeoCollapse/NeoCollapse.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../scss/variable.scss';
+
+.collapse-container {
+  .neo-collapse {
+    @include ktheme() {
+      background-color: theme('background-color');
+      color: theme('text-color');
+      border: 1px solid theme('border-color');
+      box-shadow: theme('primary-shadow');
+    }
+
+    .collapse-content {
+      &:hover {
+        @include ktheme() {
+          background-color: theme('k-primaryLight');
+        }
+      }
+    }
+  }
+
+  .active-collpase {
+    @include ktheme() {
+      background-color: theme('background-color-inverse');
+      .card-header-title,
+      .card-header-icon {
+        color: theme('text-color-inverse');
+      }
+    }
+  }
+
+  .card {
+    max-width: 100%;
+    position: relative;
+    border-radius: 0;
+    box-shadow: none;
+    @include ktheme() {
+      border-bottom: 1px solid theme('border-color');
+    }
+  }
+
+  .card-header {
+    border-radius: 0;
+    background-color: transparent;
+    align-items: stretch;
+    display: flex;
+  }
+
+  .card-header-title {
+    align-items: center;
+    display: flex;
```

Comment:
Pls use more bulma class name like `is-align-content-center`, `is-flex-grow-1`.

ref: https://bulma.io/documentation/helpers/flexbox-helpers/#align-content